webclient/dj/views.py

```diff
@@ -321,13 +322,6 @@ def post(self, request, *args, **kwargs):
             else: 
                 zaznam.pian = vytvor_pian(form.cleaned_data["katastr"])
                 zaznam.save()
-            if old_katastr.okres.kraj.rada_id != form.cleaned_data["katastr"].okres.kraj.rada_id:
-                if az.stav == AZ_STAV_ARCHIVOVANY:
-                    az.set_akce_ident(get_akce_ident(form.cleaned_data["katastr"].okres.kraj.rada_id))
-                else:
-                    az.set_akce_ident(
-                        get_temp_akce_ident(form.cleaned_data["katastr"].okres.kraj.rada_id)
-                    )
```

Comment:
ja to tam daval kvuli tomu, ze zmenen katastr muze byt v jinem regionu, ale pokud to tak nema byt (co podle bugu ne) tak hold vyhodit. :)

Reply:
Ono to má spoustu dopadů, které ošetřovat, by bylo zbytečně pracné...